docs/basic-features/image-optimization.md

```diff
@@ -163,11 +163,11 @@ Because `next/image` is designed to guarantee good performance results, it canno
 >
 > **Normalize your images**
 >
-> If you're serving images from a source that you control, consider modifying your image pipeline to normalize the images to a specific size.
+> If you're serving images from a source that you control, consider modifying your image pipeline to normalize the images to a specific size. For example, a user-provided avatar image could be cropped before being saved and then the image size would be known by the time its rendered.
 >
 > **Modify your API calls**
 >
-> If your application is retrieving image URLs using an API call (such as to a CMS), you may be able to modify the API call to return the image dimensions along with the URL.
+> If your application is retrieving image URLs using an API call (such as to a CMS), you may be able to modify the API call to return the image dimensions along with the URL. For example, set the [`loader`](/docs/api-reference/next/image.md#loader) property to match the API call expected by your CMS.
```

Comment:
This one doesn't make sense to me.

Reply:
@leerob I'm assuming that API call means the CMS has an API and thus has a way to resize images. Is that not what API means here?

Reply:
I think it needs if I'm using Contentful, for example, I can probably return the width and height in the GraphQL query. Not necessarily resize, but just get the metadata.

Reply:
Yep, @leerob's interpretatioin is what I meant in that bullet point